src/mcp_server_helm/schemas/tools.py

```
@@ -0,0 +1,642 @@
+from mcp.types import Tool
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)